deploy/helm/tests/__snapshot__/node-collector-config-map_test.yaml.snap

```diff
@@ -2408,6 +2423,13 @@ Node collector config should match snapshot when using default values:
           - k8s.istio_requests.rate
           - k8s.istio_tcp_sent_bytes.rate
           - k8s.istio_tcp_received_bytes.rate
+        experimental_metricsgeneration/istio-metrics:
+          rules:
+          - metric1: k8s.istio_request_duration_milliseconds_sum
+            metric2: k8s.istio_request_duration_milliseconds_count
+            name: k8s.istio_request_duration_milliseconds.rate
```

Comment:
This is not a rate. This is an average over the entire history of collection of this metric. I'm not sure that's what we want.